cypress/integration/seededFlows/adminFlows/users/userIndexView.spec.js

```diff
@@ -410,6 +374,86 @@ describe('User index view', () => {
               }).should('exist');
             });
           });
+
+          it('Opens the add organization modal', () => {
+            cy.enableFeatureFlag('member_index_view');
+            // Helper function for cypress-pipe
+            const click = (el) => el.click();
```

Comment:
(suggestion / not blocking) - since we need this helper a few times now, we could bring it out of the individual specs / into the top of the `describe` block, so all the tests could all use the same function

Reply:
Thank you for the suggestion! I'm going to go ahead and address this comment, as well as [this](https://github.com/forem/forem/pull/17879#discussion_r896642769) one in a separate PR. 😊